app/presenters/courses_presenter.rb

```diff
@@ -164,4 +165,26 @@ def wikidata_stats
   def creation_date
     I18n.l campaign.created_at.to_date
   end
+
+  # Returns the wiki expert username for the given course, if available
+  def assigned_wiki_expert_for_course(course_id)
```

Comment:
I'd prefer for the new code for this *not* to live in CoursesPresenter. Since this is only being used for tagged courses, it could be done within the controller.